modular-docs-manual/content/topics/introduction.adoc

```diff
@@ -2,3 +2,5 @@
 = Introduction
 
 This manual provides instructions on how to author modularly structured documentation based on user stories. The manual defines used terminology, describes components that form modular documentation, and instructs writers on how to use provided templates to turn user stories into modular documentation.
+
+IMPORTANT: As modular documentation imperatives continue to evolve, the details and recommendations in this guide might be modified. The modular documentation team recognizes that the fluctuation in recommendations can be frustrating for contributors trying to adopt modularization standards and apologizes in advance as the initiative pursues the best possible outcome for modularized documentation.
```

Comment:
Do we really need this? I think it's a given.

Reply:
I'd say it's definitely not a given, based on the comments in the repo "Issues" and the consistent feedback on the mod doc manual that I get in my repo workshops. People are very agitated on the moving target and changes in file names, heading formatting modifications, template tweaks, etc. I have to say something like this note in reply each time it comes up, so we very much need this upfront clear disclaimer to address it, imo.

Reply:
Understood. But then we're failing in educating ppl about the fact that it's a project like any other. It's even on GH. It's not set in stone. That doesn't invalidate your point, but I think we should be clear about that up front. All guidelines evolve. Even style guides are published in new revisions.

Could we perhaps cut it to just the first sentence? The rest is kind of a "don't-hate-us" apology, which, I think, shouldn't be a part of docs.

Reply:
Kept first sentence